### ISSUE TITLE
Cannot read property 'column_name' of undefined

### ISSUE THREAD

**Issue by MiniDigger** [knex/knex-schema-inspector#23]
Using a postgres db, the query should be an insert, but am just a user, I am not sure.

Stacktrace:
```
/directus $ npx -n="--trace-warnings" directus roles create --name Administrator --admin 
(node:1094) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'column_name' of undefined
    at Postgres.<anonymous> (/directus/node_modules/knex-schema-inspector/dist/dialects/postgres.js:325:51)
    at step (/directus/node_modules/knex-schema-inspector/dist/dialects/postgres.js:33:23)
    at Object.next (/directus/node_modules/knex-schema-inspector/dist/dialects/postgres.js:14:53)
    at fulfilled (/directus/node_modules/knex-schema-inspector/dist/dialects/postgres.js:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at emitUnhandledRejectionWarning (internal/process/promises.js:168:15)
    at processPromiseRejections (internal/process/promises.js:247:11)
    at processTicksAndRejections (internal/process/task_queues.js:94:32)
(node:1094) TypeError: Cannot read property 'column_name' of undefined
    at Postgres.<anonymous> (/directus/node_modules/knex-schema-inspector/dist/dialects/postgres.js:325:51)
    at step (/directus/node_modules/knex-schema-inspector/dist/dialects/postgres.js:33:23)
    at Object.next (/directus/node_modules/knex-schema-inspector/dist/dialects/postgres.js:14:53)
    at fulfilled (/directus/node_modules/knex-schema-inspector/dist/dialects/postgres.js:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(node:1094) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
    at emitDeprecationWarning (internal/process/promises.js:180:11)
    at processPromiseRejections (internal/process/promises.js:249:13)
    at processTicksAndRejections (internal/process/task_queues.js:94:32)
^C
```

for reference: https://github.com/directus/next/issues/807

**Comment by MiniDigger** [comment]
This seems to happen with any query I trigger

**Comment by codeclown** [comment]
There might be a fix for this in #19. I bumped into this bug myself when writing that PR.

**Comment by MiniDigger** [comment]
ah nice, then its not just my hyper secure corporate enterprise configuration of my db acting up, thats good to hear.
It looks like that PR is not close to be merged yet, do you think you could cherry pick the fix into its own PR?
Or maybe @rijkvanzanten can do it directly, or I could do it too
(assuming this is the fix: https://github.com/knex/knex-schema-inspector/pull/19/files?diff=unified&w=1#diff-241a92bc77a6f7efb5e272de443b7ea90213da7142d15ed3b3fe9d27d867511cR3377)
point is, I would like to get my hands on that fix rather quickly :P

**Comment by MiniDigger** [comment]
validated that fix by manually editing node_modules, ill create a PR once I am home (if rijk isn't faster)

**Comment by rijkvanzanten** [comment]
I'm down to merge #19 straight away @codeclown! Do you happen to have a Discord? Would be nice to discuss where we can take this lib together 🙂

**Comment by rijkvanzanten** [comment]
Fixed in #19
